test-site/scripts/create-verticals.js

```diff
@@ -18,27 +18,27 @@ const verticalConfiguration = {
   locations: {
     verticalKey: 'KM',
     template: 'vertical-map',
-    cardName: 'location-standard'
```

Comment:
Will we still have Percy coverage for all the single-language variants if we replace them with multi-lang here? We want to ensure total coverage for both the single and multi-language variants. Even though the multi-lang and single-lang are supposed to appear the same, they are technically different files. This means someone could bork one and not the other.

Reply:
you are right, this will only cover multilang variants. I can revert this to the default configuration for en and have some config override for es files to take multilang cards instead.